pootle/apps/virtualfolder/receivers.py

```diff
@@ -0,0 +1,13 @@
+
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from virtualfolder.models import VirtualFolder
+
+
+@receiver(post_save, sender=VirtualFolder)
```

Comment:
Please note that priorities must be recalculated when a VirtualFolder is deleted.